readme.md

```diff
@@ -24,7 +24,7 @@ Multi-platform and multi-language.
 - Bot behavior is customizable via configuration files (config as code)
 - Use ESLint-like shared config presets for ease of use and simplifying configuration (JSON format only)
 - Lock files are supported and updated in the same commit, including immediately resolving conflicts whenever PRs are merged
-- Get replacement PRs to migrate from a deprecated dependency to the community suggested replacement (npm packages only)
+- Get replacement PRs to migrate from a deprecated dependency to the community suggested replacement
```

Comment:
I think Renovate only supports replacement PRs for _some_ managers now:

- https://github.com/renovatebot/renovate/issues/14149#issuecomment-1409822662

How about something like this?

```suggestion
- Get replacement PRs to migrate from a deprecated dependency to the community suggested replacement, works with _most_ managers, see [issue 14149](https://github.com/renovatebot/renovate/issues/14149) for exceptions
```